app/github_public_repo/estimate_csv_creator.py

```diff
@@ -0,0 +1,32 @@
+import logging
+
+import pandas as pd
+from flask import current_app as app
+
+from app.database_etl.airtable_records_handler import get_all_records, airtable_get_request
+
+# Load env variables for Airtable request
+AIRTABLE_API_KEY = app.config['AIRTABLE_API_KEY']
+AIRTABLE_GITHUB_CSV_FIELDS_REQUEST_URL = app.config['AIRTABLE_GITHUB_CSV_FIELDS_REQUEST_URL']
+
+# Get columns that should be pulled into CSV
+headers = {'Authorization': 'Bearer {}'.format(AIRTABLE_API_KEY)}
+data = airtable_get_request(AIRTABLE_GITHUB_CSV_FIELDS_REQUEST_URL, headers)
+
+# Get csv columns and create csv
+try:
+    # Extract fields
+    records = data['records']
+    fields = [x['fields']['Field Name'] for x in records]
+    logging.info("Successfully retrieved field names from Airtable")
+
+    # Get estimates from Rapid Review: Estimates table for specified fields
+    csv_records = get_all_records(fields)
```

Comment:
I think get_all_records already implements error logging. With that in mind, we should move the try except to after this line IMO so we can log a more targeted error message

Reply:
It does implement error logging, but all it does is it logs the KeyError e. So if we have get all records inside this try except, and it encounters the keyerror inside this function, it will pass the exact same error back to the except and the same 'e' will be logged just as it would be inside the get all records function. Does that make sense? Basically the keyerror will bubble up back to the original try except and it will still be logged with the same granularity. See handle_airtable_error on line 36 of airtable_records_getter

Reply:
sure, it'll log twice in that scenario (a very very nitpicky concern tho, in fact probably not even worth)